package.json

```diff
@@ -24,7 +24,7 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/iobroker-community-adapters/ioBroker.samsung-community/"
+    "url": "https://github.com/mwp007/ioBroker.samsung-community/"
```

Comment:
We should remove that change or ?! :-)

Reply:
oh zes, sure :-)

Reply:
I was following a tutorial, so sorry for that